autoinstrumentation/apache-httpd/Dockerfile

```diff
@@ -20,6 +20,7 @@ FROM alpine:latest
 
 COPY --from=agent /opt/opentelemetry/agent/opentelemetry-webserver-sdk /opt/opentelemetry
 
+RUN chmod 775 -R /opt/opentelemetry/
```

Comment:
@chrlic can you confirm this change is safe.

Reply:
If this fix is not fine, I'll be happy implementing another approach but... I'll need some hints 😅

Reply:
Reading through the issue, I think you're right that this is totally fine.

Reply:
Yes this is safe. I'd do it the same way.